process/testing/winfv/capz/export-env.sh

```diff
@@ -20,10 +20,10 @@ if [ -f "${METADATAMK}" ]; then
     export KUBE_VERSION=$(grep KINDEST_NODE_VERSION= ${METADATAMK} | cut -d "=" -f 2)
     export KIND_VERSION=$(grep KIND_VERSION= ${METADATAMK} | cut -d "=" -f 2)
 else
-    export KUBE_VERSION=v1.27.11
-    export KIND_VERSION=v0.22.0
```

Comment:
this would only work as a fallback from autodetecting from `metadata.mk`, so I updated both kind and kindest/node versions there (I also checked to see there were images available for those in https://github.com/kubernetes-sigs/kind/releases)